core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -131,4 +133,37 @@ public boolean getRequiresConfirmation() {
     public boolean getRequiresPOST() {
         return false;
     }
+
+    /**
+     * Category for management link, uses {@code String} so it can be done with core dependency pre-dating the version this feature was added.
+     *
+     * @return An enum value of {@link Category}.
+     * @since TODO
+     */
+    public @Nonnull String getCategory() {
+        return Category.UNCATEGORIZED.toString();
+    }
+
+    /**
+     * Categories supported by this version of core.
+     *
+     * @since TODO
+     */
+    public enum Category {
+        CONFIGURATION,
+        SECURITY,
+        STATUS,
+        TROUBLESHOOTING,
+        TOOLS,
+        MISC,
```

Comment:
```suggestion
        MISCELLANEOUS,
```

Reply:
Invites typos in plugins.

Reply:
you an argue the same for CONFIGURATION vs CONFIG?

Reply:
No, much more common term.